packages/react/src/components/UIShell/__stories__/UIShell.stories.js

```diff
@@ -53,6 +53,7 @@ import {
 } from './AppIcons';
 
 import '../components/ui-shell.scss';
+import { SideNavFlyoutMenu } from '../components/SideNavFlyoutMenu';
```

Comment:
`'SideNavFlyoutMenu' is defined but never used. Allowed unused vars must match /^_/u`

it looks like this is an unused import?